scripts/spack/configs/macos_sonoma_aarch64/spack.yaml

```diff
@@ -15,43 +15,59 @@ spack:
     - ../defaults.yaml
     - ../versions.yaml
 
-  compilers::
-  - compiler:
-      spec: clang@=14.0.6
-      paths:
-        cc: /opt/homebrew/opt/llvm@14/bin/clang
-        cxx: /opt/homebrew/opt/llvm@14/bin/clang++
-        f77: /opt/homebrew/bin/gfortran-14
-        fc: /opt/homebrew/bin/gfortran-14
-      flags: {}
-      operating_system: sequoia
-      target: aarch64
-      modules: []
-      environment: {}
-      extra_rpaths:
-      - /opt/homebrew/lib/gcc/14
+  toolchains:
+    clang_19:
+    - spec: '%c=llvm'
+      when: '%c'
+    - spec: '%cxx=llvm'
+      when: '%cxx'
+    - spec: '%fortran=gcc'
+      when: '%fortran'
+    - spec: '%openmpi@5.0.8.clang_19'
+      when: '%mpi'
 
   packages:
     all:
-      compiler: [clang, gcc]
+      target: [aarch64]
       providers:
         blas: [openblas]
         lapack: [netlib-lapack]
         mpi: [openmpi]
         zlib-api: [zlib]
+
+    #Compiler packages
+    gcc:
+      externals:
+      - spec: gcc@15.1.0 languages:=fortran
+        prefix: /opt/homebrew/Cellar/gcc/15.1.0
+        extra_attributes:
+          compilers:
+            fortran: /opt/homebrew/Cellar/gcc/15.1.0/bin/gfortran
+    llvm:
+      externals:
+      - spec: llvm@19.1.7+clang~lld~lldb
+        prefix: /opt/homebrew/Cellar/llvm@19/19.1.7
```

Comment:
`/opt/homebrew/opt/llvm@19` might be better (more general) in case another version of llvm 19 is released